VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -132,16 +132,32 @@ private async Task<string> GetObfuscatedUsernameAsync(string username)
 
         public async Task SendMessage(Guid conferenceId, string message, string to, Guid messageUuid)
         {
+            // if participant then 'to' is Id, if admin then 'to' is username
```

Comment:
could this be added to the summary of SendMessage -> To 
 `///`

Reply:
done